### PR TITLE
init_plugin

### PR DESCRIPTION
**Pull request by RoeiCarmelMsApps**

### CODE REVIEW

File: APBabyPlayer.podspec [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                "APBaby/**/*.png"]
+
+  s.xcconfig =  { 'CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES' => 'YES',
+                  'ENABLE_BITCODE' => 'YES',
+                  'SWIFT_VERSION' => '5.1'
+              }
+
+
+  s.dependency 'ApplicasterSDK'
+  s.dependency 'ZappPlayerPlugins/Basic'
```
Comment (snooky23) [inline APBabyPlayer.podspec]: remove this line

File: APBabyPlayer.podspec [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  s.dependency 'ZappLoginPluginsSDK'
```
Comment (snooky23) [inline APBabyPlayer.podspec]: remove this line

File: APBabyPlayer.podspec [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                "APBaby/**/*.xib",
+                "APBaby/**/*.png"]
+
+  s.xcconfig =  { 'CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES' => 'YES',
+                  'ENABLE_BITCODE' => 'YES',
+                  'SWIFT_VERSION' => '5.1'
+              }
+
+
+  s.dependency 'ApplicasterSDK'
```
Comment (snooky23) [inline APBabyPlayer.podspec]: why do you need ApplicasterSDK?

File: APBabyPlayer.podspec [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  s.requires_arc            = true
+  s.static_framework        = true
+  s.swift_version           = '5.1'
+
+
+
+  s.public_header_files = 'APBaby/**/*.h'
+  s.source_files = 'APBaby/**/*.{h,m,swift}'
+
+  s.resources = [
```
Comment (snooky23) [inline APBabyPlayer.podspec:24]: Please split the resources from the actual code. currently everything is located in the same folder.

File: APBabyPlayer.podspec [hunk 5]
```diff
@@ -0,0 +1,41 @@
+Pod::Spec.new do |s|
+  s.name             = "APBabyPlayer"
+  s.version           = '0.1.0'
```
Comment (snooky23) [inline APBabyPlayer.podspec]: Please align with the old versioning.
If the old version was 5.6 please use 6.0.

File: Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec [hunk 6]
```diff
@@ -0,0 +1,41 @@
+Pod::Spec.new do |s|
+  s.name             = "APBabyPlayer"
+  s.version           = '0.1.0'
```
Comment (snooky23) [inline Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec]: Please align with the old versioning.
If the old version was 5.6 please use 6.0.

File: Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  s.requires_arc            = true
+  s.static_framework        = true
+  s.swift_version           = '5.1'
+
+
+
+  s.public_header_files = 'APBaby/**/*.h'
+  s.source_files = 'APBaby/**/*.{h,m,swift}'
+
+  s.resources = [
```
Comment (snooky23) [inline Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec]: see what I wrote you above

File: plugin-manifest.json [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+      "ZappPlayerPluginBaby",
+      "ZappPlayerPlugins"
+    ]
+  },
+  "author_name": "Alex Zchut",
+  "author_email": "a.zchut@applicaster.com",
+  "manifest_version": "5.8.0",
+  "name": "APBabyPlayer",
+  "description": "Applicaster Baby Plugable player",
+  "identifier": "APBabyPlayer-ios",
```
Comment (snooky23) [inline plugin-manifest.json:14]: You have to use the same id from berfore.

File: plugin-manifest.json [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+    "class_name": "APBabyApplicasterPlayer",
+    "modules": [
+      "ZappPlayerPluginBaby",
+      "ZappPlayerPlugins"
+    ]
+  },
+  "author_name": "Alex Zchut",
+  "author_email": "a.zchut@applicaster.com",
+  "manifest_version": "5.8.0",
+  "name": "APBabyPlayer",
```
Comment (snooky23) [inline plugin-manifest.json:12]: you have to use the same name from before

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: APBabyPlayer.podspec (hunk 3 above)
Comment (RoeiCarmelMsApps) [reply APBabyPlayer.podspec]: i took it form the origin podspec

File: APBabyPlayer.podspec (hunk 1 above)
Comment (RoeiCarmelMsApps) [reply APBabyPlayer.podspec]: done

File: APBabyPlayer.podspec (hunk 2 above)
Comment (RoeiCarmelMsApps) [reply APBabyPlayer.podspec]: done

File: APBabyPlayer.podspec (hunk 4 above)
Comment (RoeiCarmelMsApps) [reply APBabyPlayer.podspec:24]: done

File: APBabyPlayer.podspec (hunk 5 above)
Comment (RoeiCarmelMsApps) [reply APBabyPlayer.podspec]: done

File: Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec (hunk 7 above)
Comment (RoeiCarmelMsApps) [reply Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec]: done

File: plugin-manifest.json (hunk 8 above)
Comment (RoeiCarmelMsApps) [reply plugin-manifest.json:14]: i did it.  same name and identifier from previous version

File: plugin-manifest.json (hunk 9 above)
Comment (RoeiCarmelMsApps) [reply plugin-manifest.json:12]: i did it.  same name and identifier from previous version

File: APBaby/ResourceHelper/APCustomPlayerResourceHelper.h [hunk 10]
```diff
@@ -0,0 +1,76 @@
+//
+//  APCustomPlayerResourceHelper.h
+//  Pods
+//
+//  Created by Miri on 09/11/2016.
+//
+//
+
+@import UIKit;
```
Comment (snooky23) [inline APBaby/ResourceHelper/APCustomPlayerResourceHelper.h:9]: this is a .h file and not a resource, hence it should be located with all your code files.

File: APBaby/ResourceHelper/APCustomPlayerResourceHelper.m [hunk 11]
```diff
@@ -0,0 +1,92 @@
+//
+//  APCustomPlayerResourceHelper.m
+//  Pods
+//
+//  Created by Miri on 09/11/2016.
+//
+//
+
+#import "APCustomPlayerResourceHelper.h"
```
Comment (snooky23) [inline APBaby/ResourceHelper/APCustomPlayerResourceHelper.m:9]: See what I wrote above

File: APBabyPlayer.podspec [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+  s.platform                = :ios, '10.0'
+  s.ios.deployment_target   = "10.0"
+  s.requires_arc            = true
+  s.static_framework        = true
+  s.swift_version           = '5.1'
+
+
+
+  s.public_header_files = 'APBaby/**/*.h'
+  s.source_files = 'APBaby/**/*.{h,m,swift}'
```
Comment (snooky23) [inline APBabyPlayer.podspec:22]: Please remove the resource helper folder since you don't point to it inside your podspec

**Review by snooky23 — CHANGES_REQUESTED** [review]

**Review by snooky23 — APPROVED** [review]

**Pull request merged**